script/override_utils.py

```diff
@@ -126,3 +127,31 @@ def override_scope_variable(scope,
             setattr(scope, name, original_value)
         else:
             delattr(scope, name)
+
+
+def call_function_get_frame(func, *args, **kwargs):
```

Comment:
@cdesouza-chromium do you think this is an okay approach?

Reply:
cc: @goodov

Reply:
Yeah, I think goodov is better qualified than I am to answer that.

Reply:
@goodov any thoughts on whether this is something we actually want to do?

I've added it, because I basically want to add code to the end of a function with access to all its variables (rather than just an override) - however, it seems fairly far down the path of hackery.

Reply:
I'm happy to move more of this stuff into a patch if we prefer that, but I think this might be quite useful for doing more work in Chromium overrides than patches.